package.json

```diff
@@ -1,15 +1,18 @@
 {
   "name": "browserama",
-  "version": "3.1.1",
+  "version": "3.2.0",
   "description": "A browser detection package that uses feature detection instead of user agent",
   "main": "out/browserama.min.js",
   "files": [
     "index.js",
     "out"
   ],
+  "publishConfig": {
```

Comment:
technically i don't think this is needed but it's probably good to have. i think you only need this for private packages.